__tests__/Description.test.js

```diff
@@ -2,7 +2,7 @@ import Enzyme, { shallow, render, mount } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import React from 'react';
 import ReactDOM from 'react-dom';
-import Description from '../../client/components/Description.jsx'
+import Description from '../client/components/Description.jsx'
 
 Enzyme.configure({ adapter: new Adapter() });
 
```

Comment:
Maybe consider deleting the extra line here.

Reply:
Also, on lines 11-13, does this need to be spread across multiple lines? I'm not sure what is standard practice for wrappers.

Reply:
Fixed!